caffe2/python/workspace.py

```diff
@@ -55,6 +55,7 @@
 operator_tracebacks = defaultdict(dict)
 
 is_asan = C.is_asan
+build_options = C.get_build_options()
```

Comment:
I started putting some build information at https://github.com/caffe2/caffe2/blob/master/caffe2/python/build.py#L23. Can we merge them together?